client/packages/host/src/components/AppDrawer/AppDrawer.tsx

```diff
@@ -151,13 +153,15 @@ export const AppDrawer: React.FC = () => {
   }, [isMediumScreen]);
 
   const onHoverOut = () => {
+    if (isMobile) return;
```

Comment:
It actually took me a little while to understand why this was necessary, but it's because "hover" events aren't registered properly on mobile, right? (They fire onClick).

So maybe just add a comment here before this line:
```ts
// Hover events not applicable on mobile devices
```